### ISSUE TITLE
Marketplace Themes: Cart redirect to Marketplace Thank you page

### ISSUE THREAD

**Issue by WBerredo** [Automattic/wp-calypso#74295]
Marketplace Themes, after the purchase, should be redirected to the Marketplace Thank you page. See pekYwv-Pz-p2.
The steps to achieve this are: 
 - [ ] Remove the `redirect_to` parameter from the cart URL for Marketplace themes.
 - [ ] Make Marketplace themes enter in the following condition (see https://github.com/Automattic/wp-calypso/issues/73838#issuecomment-1458339666): 

 
https://github.com/Automattic/wp-calypso/blob/3d1006d1780d61012d7a1f1572abd6660cbb6a7d/client/my-sites/checkout/get-thank-you-page-url/index.ts#L526-L535

**Comment by epeicher** [comment]
Backend changes that will populate the `extra` object that will be used by the above conditions will be done as part of the task: 
- https://github.com/Automattic/wp-calypso/issues/74371

**Comment by gcsecsey** [comment]
> Remove the redirect_to parameter from the cart URL for Marketplace themes.

@WBerredo @epeicher in https://github.com/Automattic/wp-calypso/pull/72837 a feature was added recently that checks the theme activation status. This is enabled via the `sync-active-theme` query parameter and provides visual feedback on the Atomic transfer.

https://github.com/Automattic/wp-calypso/blob/5c36d4f561fc711ac797bb874a6485a8304c72ea/client/state/themes/actions/add-external-managed-theme-to-cart.tsx#L130-L132

For now, I've only removed the redirect parameter for Atomic sites, not to interfere with this existing functionality. I'm not sure if we can currently check the theme activation status on the new thank you page. Maybe we should add a follow-up ticket to remove the redirect for non-Atomic sites only once we have that functionality.
